app/src/main/java/cc/ioctl/hook/FileRecvRedirect.java

```diff
@@ -53,7 +63,7 @@
     private Field TARGET_FIELD = null;
 
     private FileRecvRedirect() {
-        super(new int[]{DexKit.C_APP_CONSTANTS});
+        super(SyncUtils.PROC_ANY);
```

Comment:
```suggestion
        super(SyncUtils.PROC_ANY, new int[]{DexKit.C_APP_CONSTANTS});
```
下面有用到 `DexKit.C_APP_CONSTANTS` 的，初始化参数留着